File: PCarpet/Models/payment.cs
using PCarpet.DTO;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace PCarpet
{
    public partial class payment
    {
        public payment(PaymentDTO paymentDTO)
        {
            this.quantity = paymentDTO.quantity;
            this.username = paymentDTO.username;
            this.id_stop = paymentDTO.id_stop;
        }

        public static PaymentDTO toPaymentDTO(payment payment)
        {
            return new PaymentDTO(payment.id, payment.quantity, (int)payment.id_stop);
            //TODO
            //this.slotAddress = payment.slotAddress;
        }
    }
}

File: FrontEndPCarpet/src/models/Payment.ts
export class Payment {

    id: number;
    quantity: number;
    id_stop: number;
    username: string;

    constructor(id: number, quantity: number, id_stop: number, username: string) {
        this.id = id;
        this.quantity = quantity;
        this.id_stop = id_stop;
        this.username = username;
    }
}
